app/controllers/application_controller.rb

```diff
@@ -58,4 +58,10 @@ def redirect_to_primary_host
 
     redirect_to [request.protocol, ENV["PRIMARY_HOST"], request.fullpath].join, allow_other_host: true
   end
+
+  def check_maintenance_mode
+    if FeatureFlags.enabled?(:maintenance_mode, without_session_data: true)
+      redirect_to "/500"
```

Comment:
If the site is in maintenance mode and I visit `check-client-blah...gov.uk/privacy`, this will show me the error page but it will do it by sending me to `check-client-blah...gov.uk/500`. Which is a bit weird, because this isn't a "500" situation - 500 means there's been an error. And it's also annoying to be redirected, because if it's just a temporary blip I'd like to be able to reload the page to see if it's fixed, and reloading won't change the URL back to `.../privacy`.

So I'd recommend that instead of a redirect, we do a render, i.e. (I think) `render file: "public/500.html", status: : service_unavailable, layout: false`

Reply:
Had to go with this:

`render file: "public/500_Service_Unavailable.html", status: :service_unavailable, layout: "layouts/500"`

- If I set `layout: false`, we get none of the CSS stylings. 
- If I set `layout: true` , we get two headers and footers. 
- Therefore created another 500 page and added in a specific layout for it.

Reply:
Ignore this comment - we (as per slack message) updated CSP settings.